docs/discover/save-search.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[save-open-search]]
-== Save a search for reuse
+== Save a Discover session for reuse
 
-A saved search is a convenient way to reuse a search
+A saved Discover session is a convenient way to reuse a search
 that you've created in *Discover*.
-Saved searches are good for adding search results to a dashboard,
+Discover sessions are good for saving a configured view of Discover to use later or adding search results to a dashboard,
 and can also serve as a foundation for building visualizations.
 
```

Comment:
```suggestion
+
TIP: Until Elastic Stack version 8.18.0 Discover sessions were called "saved searches".
+ 
```
I'd say an info callout providing the information about the renaming on our `Save a Discover session for reuse` should be sufficient, wording could of course be better, wdyt @wajihaparvez

Reply:
we can take care of this in a follow up issues, to unblock this